data.json

```
@@ -521,4 +521,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
   }
```

I think we need a comma here
```suggestion
  },
```